neurons/validator.py

```diff
@@ -176,33 +172,39 @@ async def forward(self):
                 deserialize=False,
                 timeout=300
             )
-            # Getting the responses
+            # Processing miner output into rewards
             for j in range(0, len(responses)):
                 if responses[j].miner_response is not None:
                     if task_string.endswith('speech'):
                         miner_output_data = audio_decode(responses[j].miner_response)
                     else:
                         miner_output_data = responses[j].miner_response
-                        
-                    bt.logging.info(f'DECODED OUTPUT DATA: {miner_output_data}')
                     
-                    successful.append([miner_output_data, j])
+                    results.append(
+                        int(self.process_validator_output(
+                            miner_output_data,
+                            sample_request['output'],
+                            task_string
+                        ) * 1000) # 'numpy.float64' object cannot be interpreted as integer
+                    )
                 else:
-                    bt.logging.warning(f"Miner {j} failed to respond.")
```

Comment:
You don't want to log this?  Might be helpful later.

Reply:
`j` here is the miner's ID in the for loop, not the miner's UID in the network. I got rid of it because it gives us false info if we're looking at the logs.